tensorly/datasets/nn_parafac_IL2_example.py

```diff
@@ -0,0 +1,89 @@
+"""
+Non-negative PARAFAC Decomposition of IL-2 Response Data
+==============================
+
+Here we will provide an example of how to use non-negative PARAFAC tensor 
+decomposition (:func:`tensorly.decomposition.parafac`) to both reduce the dimensionality 
+of a tensor of experimental data, as well as make insights regarding the underlying structure 
+of that data.
+
+To do this, we will work with a tensor of experimentally measured cell signaling data.
+"""
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+from tensorly.decomposition import non_negative_parafac
+from imports import IL2data
+
+#%%
+# Here we will load a tensor of experimentally measured cellular responses to 
+# IL-2 stimulation. IL-2 is a naturally occurring immune signaling molecule 
```

Comment:
Might be useful for non-experts to have a slightly longer intro, e.g. what kind of signalling does it do, etc?
This example is super interesting for non experts too!